src/main/java/it/gov/pagopa/pu/sil/mapper/NativeAssessmentsBalanceMapper.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.pu.sil.mapper;
+
+import it.gov.pagopa.pu.classification.dto.generated.AssessmentsBalanceView;
+import it.gov.pagopa.pu.sil.dto.generated.BalanceDTO;
+import it.gov.pagopa.pu.sil.dto.generated.DebtPositionTypeOrgDTO;
+import it.gov.pagopa.pu.sil.dto.generated.SectionDTO;
+import it.gov.pagopa.pu.sil.dto.generated.AssessmentDTO;
+import org.springframework.stereotype.Service;
+
+@Service
+public class NativeAssessmentsBalanceMapper {
```

Comment:
I would omit `Native`, lets use the prefix just for Legacy ;)
```suggestion
public class AssessmentsBalanceMapper {
```